src/main/resources/liquibase.xml

```diff
@@ -356,4 +356,34 @@
             </column>
         </createTable>
     </changeSet>
+
+    <changeSet id="202004101530" author="apoorva" context="POC">
+        <dropColumn tableName="patient">
+            <column name="first_name" type="VARCHAR(100)">
+                <constraints nullable="false" unique="false"/>
+            </column>
+        </dropColumn>
+        <dropColumn tableName="patient">
+            <column name="last_name" type="VARCHAR(100)">
+                <constraints nullable="true" />
+            </column>
+        </dropColumn>
+        <dropColumn tableName="patient">
+            <column name="date_of_birth" type="DATE">
+                <constraints nullable="true" />
+            </column>
+        </dropColumn>
+        <addColumn tableName="patient">
+            <column name="name" type="VARCHAR(100)">
+                <constraints unique="false"/>
+            </column>
```

Comment:
varchar 100 is enough? firstname alone had 100 :)

Reply:
Thought so, let me know If you want to keep it more.

Reply:
https://www.liquibase.org/documentation/changes/rename_column.html won't work?

Reply:
Will work, but do we want to rename anything here?

Reply:
firstname to name, that way we will at least have firstname in the name column no?

Reply:
Sure if we want to keep that data we can do that.

Reply:
Thanks
We need some name, otherwise existing users will be ` ` Blank users :)